src/SpendPermissionManager.sol

```diff
@@ -153,14 +154,19 @@ contract SpendPermissionManager is EIP712 {
 
     /// @notice Approve a spend permission via a signature from the account.
     ///
+    /// @dev Compatible with EIP-6492 signatures (https://eips.ethereum.org/EIPS/eip-6492)
```

Comment:
can we also add a `@dev` tag to explicitly call out that accounts are automatically deployed if `initCode` is present?

Reply:
Also EIP/ERC is a non-obvious distinction, but the current culture is to call most things ERCs instead of EIPS now so "ERC-6492". Link can stay as-is though.

Reply:
oh interesting! I didn't know that but I like this - basically trivial distinction in the first place